api/hooks/discordBot/commands/sdtd/top.js

```diff
@@ -13,7 +13,9 @@ class Top extends Commando.Command {
         key: 'type',
```

Comment:
So one thing I noticed while testing this:
![image](https://user-images.githubusercontent.com/22315101/95651803-01e14700-0aed-11eb-98d3-ae75bce6df7e.png)

If a user types `Deaths`, it errors out as you say. However, I think CSMM should recognize it and 'fix' the argument for the user. So a `.toLowercase()` in the validation would be nice. Make sure the actual value of the arg later in the command is also lowercased though.

Possible the [ArgumentType#parse method](https://discord.js.org/#/docs/commando/master/class/ArgumentType?scrollTo=parse) can be used for this.

Reply:
So, if I understand you correctly, you want commands to be case insensitive and also make the whole command lowercase for later usage?

Reply:
I think the best approach is to let the validation be case insenstive. (Deaths - OK, deaths - OK, deths - Not OK) and at the start of the command use `.toLowercase()` so that the command does not error when it gets to the DB sorting